ts/features/messages/components/MessageAttachmentPreview.tsx

```diff
@@ -33,11 +33,19 @@ const styles = StyleSheet.create({
   }
 });
 
-const renderFooter = ({ attachment, path }: MvlDownload) =>
+const renderFooter = (
+  { attachment, path }: MvlDownload,
+  onShare?: () => void,
+  onOpen?: () => void,
+  onDownload?: () => void
+) =>
   isIos ? (
     <FooterWithButtons
       type={"SingleButton"}
       leftButton={confirmButtonProps(() => {
+        if (onShare) {
+          onShare();
```

Comment:
```suggestion
          onShare?.();
```
If you like a shorter syntax, optional chaining could be effective

Reply:
@Undermaken you know what? I tried to do it several times, but I missed the `.` and it never compiled 😅

Reply:
This syntax is a little bit tricky to remember
Indeed I bookmarked this link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining 😁